### ISSUE TITLE
Binding both legend and chart

### ISSUE THREAD

**Issue by protaeus** [vega/vega-lite#5692]
I am using vega lite 4 through altair, which now possesses ability to bind the legend whereby making it interactive. I noticed that once one binds the legend, the interactivity with the chart no longer works. I was hoping it would be possible to bind both the legend and chart so that it works reciprocally. I posted an inquiry with altair (https://groups.google.com/forum/?utm_source=digest&utm_medium=email#!topic/altair-viz/c4Xk8htgh-s) and was recommended to submit the request to vega since it appears there is no way to accomplish this task (at least without going the round-about way by creating the legend as a chart).

To summarize my request, I would like to create a chart that gives the end-user the ability to select either the chart or legend without having to go the round-about way of creating a legend as a chart. The primary downside of creating a legend as a chart is that it removes the ability to employ the configuration options that go along with the legend (e.g. columns).

**Comment by arvind** [comment]
Hi @protaeus, thanks for the question! Yes, by default, we disable any direct manipulation interaction (e.g., clicking on the chart) when a selection is bound to HTML widgets or legends (our reasoning is described in #5277). To re-enable this type of interactivity, you can use the `on` and `clear` properties of a selection. For instance, here's an updated selection definition for our interactive legend example ([see it live in the editor](https://vega.github.io/editor/#/url/vega-lite/N4KABGBEAkDODGALApgWwIaQFxUQFzwAdYsB6UgN2QHN0A6agSz0QFcAjOxge1IRQyUa6ALQAbZskoAWOgCtY3AHaQANOCgB3RgBMW2MAGYADMfURIKRtXwGATKfNQd6PJhzBIrAE5iDkFzdSViU0QjFuAE9UZCU8EXR4b25YWBFGJR1WWDxvRmRYeUUVAF8nSAxvAGt-dG9kTHLYZDFkeDweFQ8NCwysnO9Ig1AIUag8SMJkf1RWMQ61KAAzfLEdWAMAbUhmvILIAF1VKHY+-1bqWJ1FntHIZX94CXga46gnhu9-HXYxJ8YXpBbiUNGUNJBYvBuDoMtRhrdIAAPeFjCwdGIAVSUzH8kU+qGU+icY0gKxa1xwAVc02JdwmU38eDC3G86D8tIs6ERjA2HgC3AwGQMSzZzWOpJZGDw-gApABNRaQDovADKjAAXtMcMYQWMwSSht1UVB0NRqPVaEz-LBWKg1LcLGS1o9uCFpRzxpMtVAAI6sdBxZiuRhUe3GnZuQGU+CxJlfD2QLk8gxKOZiYG0yBQiJfI2o0mrClQXb5DYJ+neyBKAUZNlh-MINnezz8NCV+DU6gsyIOdiQXWjfV3biERLMQ1gEb5qGZZidYY7FptDoPSl9bK5Ibiihs1jegCMQ5JO7Ee4MxjodmBoJAJSAA)).

```json
"selection": {
    "industry": {
      "type": "multi", "fields": ["series"], "bind": "legend",
      "on": "click", "clear": "dblclick"
    }
  },
```

**Comment by protaeus** [comment]
Hi Arvind,

Thank you for the update. Yes - it worked like a charm in my code. Woohoo!

John

**Comment by lassepe** [comment]
Quick followup on this problem:

The suggestion of @arvind works like a charm for a single view. However, if I have a concatenation of multiple views then this only works for the first view.

For example, if I `hconcat` a pie chart with a bar chart, I can click on the legend or the pie-elements to highlight different categories. However, interaction with the bar chart does not work.
[See this live demo, which demonstrates this problem](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JACyUAVhDYA7EABoQAdxoATemgDMABh1L4NMljQAmPUrVJMKVKAYAnQWhBWbBBrMQ5BbAJ4IcLKY-EhQDmwQEPw0smoMEJgONHAQUjLyAL4GUHJQ1mgA2qDIDgDWLkgOUIogEHCCcFCY7PJ2ILHxiQ5+aKCYfjhwLhCxZI21AGYpgmoQRXVwyakgALpKTJ0ujWRBarVyLlDCUBVKx3BVLmpMgsc0pyCZ2SBBuWpjfSD0cDZfSGQyA4+JhhugIAwEFMZvt0LlPJhagMhi4AI4MJDBOjWGikWqJMIVOFBUEOJ7nNi+MntaYNWGLZbzJTIsEgWRsBCxJDOJTQblg0BcRCs-KgshsHrmJhPF5sHBhOi9dq5OJ0VpfeqNZrq9CdBJJXpKYjchhggCML2NglNaB0lFMz2yJSqRJATCuSjebA+sjIXyVoABQJBrIhUKUtNmRzYCKRg1Z6MxLRsLTxvJsj2JwSW5JAAA8vpH6fVGXGUeh2ZzZNz8fkJu0hYEjtZyBK-FKZUo5QqBl8VR8WocGw0moO2h04vqerUrTbUBajSawXaHc9VpkgA).
